addons/test/misra/misra-test.c

```diff
@@ -1491,8 +1494,8 @@ static void misra_15_7(void) {
     }   // no-warning
   }
 
-  if (a==2) {} else if (b==4) {} // 15.7
-  if (a==2) {} else { if (b==4) {} } // no-warning
+  if (a==2) {} else if (b==4) {} // 15.7 config
```

Comment:
It's good that you do detect and warn about these. I feel these `config` messages are not what we wanted to test here. either if you update the test code so there are not unknown symbols.. or I do it later..